go/porcelain/deploy_test.go

```diff
@@ -287,7 +287,7 @@ func TestUploadFiles_Cancelation(t *testing.T) {
 	for _, bundle := range files.Files {
 		d.Required = append(d.Required, bundle.Sum)
 	}
-	err = client.uploadFiles(ctx, d, files, nil, fileUpload, time.Minute)
+	err = client.uploadFiles(ctx, d, files, nil, fileUpload, time.Minute, false)
```

Comment:
Wondering if we could add a test here which exercises the new code paths we introduced (i.e. checking we don't retry `400` and `422` errors when uploading).

Reply:
@JGAntunes do you have any pointers for the testing? I seem to be a bit stuck here. We are also having trouble running the test suite locally, which is slowing this down a bit 😬

Reply:
about running the test suite locally, if it seems like nothing is running it's probably because of the retries loop.

try running the tests as:
```
 go test -race github.com/netlify/open-api/v2/go/porcelain --run TestUploadFiles400Errors -v
```

And you should get some output (hopefully)

Reply:
Thanks for the tip @4xposed!! We got the tests running and can see the output which has been very helpful :D